### PR TITLE
feat(RELEASE-1244): make create-github-release task idempotent

### PR DESCRIPTION
**Pull request by jinqi7**

### CODE REVIEW

File: tasks/create-github-release/tests/test-create-github-exist-release.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-create-github-release
```
Comment (johnbieren) [inline tasks/create-github-release/tests/test-create-github-exist-release.yaml]: this should match the file name

File: tasks/create-github-release/tests/test-create-github-exist-release.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+spec:
+  description: |
+    Run the create-github-release task
```
Comment (johnbieren) [inline tasks/create-github-release/tests/test-create-github-exist-release.yaml]: update this please

**Review by johnbieren — COMMENTED** [review]
@ernesgonzalez33 the code here LGTM, but can you also give it a quick look as you were involved in the creation of the release to github pipeline?

File: tasks/create-github-release/tests/test-create-github-exist-release.yaml (hunk 2 above)
Comment (jinqi7) [reply tasks/create-github-release/tests/test-create-github-exist-release.yaml]: Updated, thanks!

**Comment by johnbieren** [comment]
LGTM but I'd like to wait for Ernesto to give it a look before we merge

**Comment by ergonzal22** [comment]
> @ernesgonzalez33 the code here LGTM, but can you also give it a quick look as you were involved in the creation of the release to github pipeline?

@johnbieren LGTM as well! Nice idea

**Review by johnbieren — APPROVED** [review]

**Pull request merged**